doc/iterations/2024.11/add_undo_redo_support.adoc

```diff
@@ -38,6 +38,34 @@ Pressing ctrl+y will redo the action that was previously undone.
 
 This should work regardless of the representation.
 
+=== Scenario (serialized representation)
+
+Serialized representation currently used in Sirius Web contains a layout part that is not contained in an EMF resource.
+
+On an unsynchronized representation the same scenarios as above should work.
+
+1. Several users are working on the same node 1
+- An user u1 resize a node from 10*10 to 100*100
+- An user u2 add sub nodes to the same node
+- If An user u1 press ctrl+z
+- *Expected*: the node will be not be resize to 10*10 if the sub nodes added by u2 do not fit inside, it will be resized to a default size.
```

Comment:
Yes but it's a very subtle example because we will try anyway to resize it to 10*10, it just happens that we won't succeed. It's a very good example to keep in mind but it's a complex one. Try to start with an example that does not involve diagrams to have a bird eye view of the problem. The solution cannot be specific to diagrams.

Consider this, one user create a portal, adds three forms inside. Another user adds a fourth forms and then the first users undo the addition of the third forms. There will only be the forms 1, 2 and 4.